uberenv.py

```diff
@@ -361,16 +375,26 @@ def setup_paths_and_dirs(self):
             if not uberenv_plat is None:
                 self.spack_config_dir = pabs(pjoin(spack_configs_path,uberenv_plat))
 
-        # Spack local packages path
-        spack_packages_path = pabs(pjoin(self.uberenv_path,"packages"))
-        if "spack_packages_path" in self.project_opts.keys():
-            new_path = self.project_opts["spack_packages_path"]
-            if new_path is not None:
-                spack_packages_path = pabs(new_path)
-        # All local packages
-        self.pkgs = pjoin(spack_packages_path, "*")
+        # Find project level packages to override spack's internal packages
+        self.spack_packages_base_paths = []
+        if "spack_packages_base_paths" in self.project_opts.keys():
+            # packages directories listed in project.json
+            _paths = self.project_opts["spack_packages_base_paths"]
```

Comment:
Not sold on this option name but wanted to show that it wasn't the individual packages directory ("packages" vs "packages/axom, packages/mfem, etc...")

Reply:
I relied on the plural form:
spack_packages_path for all packages,
spack_package_path for a specific package.

Reply:
> I relied on the plural form:
> spack_packages_path for all packages,
> spack_package_path for a specific package.

so `spack_packages_paths`? since its the packages directory and there are multiples of them? This already confuses me and I did it.

Reply:
That works for me. With the weakness that every "s" counts.

Reply:
spack_packages_path_list?

Reply:
I don't have strong opinions anymore at this point :)

Reply:
I'm assuming that the order of the paths matters as well, right?

Reply:
Is there a evoke `$PATH` or `PYTHONPATH` -- i.e. variables with lists of directories where the order matters?

Maybe `spack_packages_path` would suffice if we make the connection in the docs?

(Somehow `spack_packages_path_list` is too specific in number of characters, but not specific enough about the order being important)